numpy/doc/glossary.py

```diff
@@ -377,6 +380,15 @@
    structured data type
        A data type composed of other datatypes
 
+   subarray
+       A :term:`structured data type` may contain a :term:`ndarray` with its
+       own dtype and shape.
```

Comment:
I wonder if instead we want to use the term "subarray data type", giving `np.dtype((np.float_, 3))` as an example.

Reply:
ok

Reply:
adopted, with a more extensive example